pkg/token/server.go

```diff
@@ -289,6 +289,37 @@ func newHandlerFunc(d *daemon, timeout time.Duration) http.Handler {
 		io.WriteString(w, string(response))
 	}
 
-	// timeout handler
-	return http.TimeoutHandler(http.HandlerFunc(mainHandler), timeout, "Handler timeout by token-server-timeout")
+	// logging && timeout handler
+	return withLogging(http.TimeoutHandler(http.HandlerFunc(mainHandler), timeout, "Handler timeout"))
+}
+
+// withLogging wraps handler with logging
+func withLogging(handler http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		startTime := time.Now()
+		log.Infof("Received request: Method[%s], URI[%s], RemoteAddr[%s]", r.Method, r.RequestURI, r.RemoteAddr)
```

Comment:
can you help to add some request ID to allow tracing the request in the log?
example: https://medium.com/@funfoolsuzi/logging-with-request-id-in-go-microservice-21485c6730da

Reply:
sorry but I think the requested id should be listed in the back as it is not he main point in the log

Reply:
its just a sample below:
```sh
INFO[2024-02-27T14:30:25+09:00] Response sent: StatusCode[200], Latency[96.108125ms], requestId: c9652f69-3356-43c0-84c5-41aec730046a
```

Reply:
@mlajkim fixed in 

```bash
INFO[2024-02-27T15:53:54+09:00] Received request: method[POST], URI[/roletoken], remoteAddr[127.0.0.1:63159] requestID[806953c0-2e57-4d22-a148-4053dd546f96] 
INFO[2024-02-27T15:53:54+09:00] Role token cache miss, successfully updated role token cache: target[{yby.taniwa.provider.example:role.provider-client,,0,0}] 
INFO[2024-02-27T15:53:54+09:00] Response sent: status code[200], latency[92.40825ms], requestID[806953c0-2e57-4d22-a148-4053dd546f96]

Reply:
lgtm

Reply:
@t4niwa 
mb but was wondering if you can do the following instead (as it looks better to start with verb)
```
INFO[2024-02-27T15:59:47+09:00] Successfully updated role token cache due to a cache miss: target[{domain:role.dummy,,0,0}] 

```

Reply:
@t4niwa 
I am sorry but
instead of `URI` I think `ENDPOINT` makes more sense.
Sorry for unorganized comments.

Reply:
@mlajkim thank you
> mb but was wondering if you can do the following instead (as it looks better to start with verb)

fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/83/commits/a3ec3271ea8307c313ce21de704b8fa1175505d5

> instead of URI I think ENDPOINT makes more sense.

fixed in https://github.com/AthenZ/k8s-athenz-sia/pull/83/commits/d689cd82c3bf1ff217f727c9ff423eb7381de536

Reply:
@t4niwa ty I am good now.

Reply:
LGTM👍